src/main/java/com/xpanxion/java/springboot/da1/demo/repository/student2/MemberRepository2.java

```diff
@@ -8,4 +9,7 @@
 
 @Repository
 @Transactional
-public interface MemberRepository2 extends JpaRepository<Member2, Integer> { }
+public interface MemberRepository2 extends JpaRepository<Member2, Integer> {
+
+    Member2 findById(int id);
```

Comment:
I can't be sure, is this necessary?  I think `Member2 findById(int id);` can be removed.  Can you verify?

Reply:
It ends up throwing an error when I deleted it.

Reply:
ok